joy_teleop/scripts/joy_teleop.py

```diff
@@ -269,7 +271,7 @@ def get_service_type(self, service_name):
         return self.service_types[service_name]
 
     def update_actions(self, evt=None):
-        for name, cmd in self.command_list.iteritems():
+        for name, cmd in six.iteritems(self.command_list):
```

Comment:
Is this PR meant to anticipate for Noetic? If so why using `six`? Wouldn't it be clearer to branch and use plain Python3?
Note that those small changes were already done for Dashing.
The same comment applies to `mouse_teleop.py`.